src/transaction.js

```diff
@@ -299,7 +314,7 @@ class Transaction {
     }
 
     return this._firestore
-      .request('beginTransaction', request, /* allowRetries= */ true)
+      .request('beginTransaction', request, this._requestTag, true)
```

Comment:
A simple trick to retain the comment is to declare a file level constant like `const allowRetries_ = true;` and then pass it instead of `true`.

Reply:
I still think this can help improve readability. But take it as a nit.

Reply:
Sorry, I missed this in the original round of feedback. I have fixed this now. Thanks for being on top of things!